lib/printers/pretty.js

```diff
@@ -22,30 +22,47 @@ class PrettyPrinter {
     }
 
     let warningCount = this.options.quiet ? 0 : results.warningCount;
-    let count = results.errorCount + warningCount;
+    let todoCount = this.options.quiet || !this.options.includeTodo ? 0 : results.todoCount;
```

Comment:
I cannot understand this conditional (using boolean logic && and ternary in the same statement). Can you break it out into separate variables then do the boolean logic on that?

Reply:
Done.